src/niswitch/metadata/functions_addon.py

```diff
@@ -86,8 +86,12 @@
 
 # Default values for method parameters
 function_default_value = {
-    'InitWithOptions':  { 'parameters': { 1: { 'default_value': False, },
-                                          2: { 'default_value': False, },
-                                          3: { 'default_value': '', }, }, },
+    'InitWithOptions':   { 'parameters': { 1: { 'default_value': False, },
```

Comment:
you can mark this function so code-gen is skipped. It's essentially dead code now, right?